DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: urltools
 Type: Package
 Title: Vectorised Tools for URL Handling and Parsing
-Version: 1.5.1
+Version: 1.5.1.9000
```

Comment:
Do you mind if we make a 1.5.2 with this PR change and we make a "release" here on github? That way I can refer to it on `devtools::install_github` and stop bothering you to send this to CRAN sooner